carbonmark/lib/constants.ts

```diff
@@ -140,6 +146,8 @@ export const urls = {
   docs: "https://docs.carbonmark.com",
   docsResourcesFees:
     "https://docs.carbonmark.com/get-started/understanding-fees-on-carbonmark",
+  payViaBankForm:
+    "https://api.hsforms.com/submissions/v3/integration/submit/26010207/2f87cd63-f8a7-43e9-9483-ac541a614762",
```

Comment:
Maybe not a risk but should this be a vercel secret perhaps?

Reply:
@0xemc Good point, and honestly I'm not sure if there is any risk to exposing this url publicly, maybe @Atmosfearful has some thoughts on this? Happy to move it to a vercel variable if we feel it's necessary.

Reply:
meh they are intended to be public unauthenticated urls anyways, sniffable from our site payload, only benefit of env vars would be hiding from github bots I guess🤷